lib/Driver/UnixToolChains.cpp

```diff
@@ -341,6 +350,8 @@ toolchains::GenericUnix::constructInvocation(const DynamicLinkJobAction &job,
     llvm::sys::path::append(linkFilePath, "static-executable-args.lnk");
   } else if (staticStdlib) {
     llvm::sys::path::append(linkFilePath, "static-stdlib-args.lnk");
+  } else if (embeddedEnabled) {
```

Comment:
Do we need to fix SwiftDriver in a similar way?

Reply:
yea; I have a patch for that almost ready

Reply:
https://github.com/apple/swift-driver/pull/1471